config/main.py

```diff
@@ -3312,5 +3432,354 @@ def autorestart(container_name, autorestart_status):
 
     config_db.mod_entry('CONTAINER_FEATURE', container_name, {'auto_restart': autorestart_status})
 
+#
+# 'vxlan' group ('config vxlan ...')
+#
+@config.group()
+@click.pass_context
+def vxlan(ctx):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
```

Comment:
Will these CLI commands configured/enabled frr configurations?

Reply:
No. These only modify the VXLAN_TUNNEL, VXLAN_TUNNEL_MAP, EVPN_NVO, VRF tables.